prql-compiler/tests/integration/queries/arithmetic.prql

```diff
@@ -0,0 +1,27 @@
+[
+    { x_int =  13, x_float =  13.0, k_int =  5, k_float =  5.0 },
+    { x_int = -13, x_float = -13.0, k_int =  5, k_float =  5.0 },
+    { x_int =  13, x_float =  13.0, k_int = -5, k_float = -5.0 },
+    { x_int = -13, x_float = -13.0, k_int = -5, k_float = -5.0 },
```

Comment:
I'm guessing this isn't auto-formatted? It's beautiful! 

(Maybe that's a side project — align relation literals like this automatically!)